upgrade/task_hcc_upgrade_management_node.adoc

```diff
@@ -24,7 +24,11 @@ NOTE: The vCenter Plug-in 4.4 or later requires a management node 11.3 or later
 
 Choose one of the following management node upgrade options:
 
-NOTE: There are no additional functionality changes or bug fixes in management node 12.3.1. If you are already running management node 12.3, you do not need to upgrade it to 12.3.1.
+[NOTE]
+====
+* Management node 12.3.2 contains a security mitigation for storage clusters with the Virtual Volumes (VVols) feature enabled. If your storage cluster is already at Element 12.3 and the VVols feature is enabled, upgrading to 12.3.2 is highly recommended.
+* There are no additional functionality changes or bug fixes in management node 12.3.1. If you are already running management node 12.3, you do not need to upgrade it to 12.3.1.
```

Comment:
Should these 12.3.1 mentions be changed to 12.3.2?

Reply:
I left them that way to align with the release notes.
The release notes contain both the 12.3.1 and the 12.3.2 improvements.
12.3.2 is specific to storage clusters already at Element 12.3 with VVols enabled. I could change the 12.3.1 to 12.3.x.
what do you think?

Reply:
Ah, I see - if the RNs cover both 12.3.1 and 12.3.2, it might make more sense to leave it as is.  I'll close this out.  Carry on!